src/id5-api.js

```diff
@@ -29,8 +29,8 @@ ID5.init = function (options) {
     utils.logInfo(`ID5 detected referer is ${referer.referer}`);
 
     const storedResponse = JSON.parse(utils.getCookie(cfg.cookieName));
-    const storedDate = new Date(+utils.getCookie(lastCookieName(cfg)));
-    const refreshNeeded = storedDate.getTime() > 0 && (Date.now() - storedDate.getTime() > cfg.refreshInSeconds * 1000);
```

Comment:
if there was no cookie found, then `storedDate.getTime()` was `0`, which resulted in this evaluating to `false`

Reply:
agreed